state-chain/pallets/cf-swapping/src/lib.rs

```diff
@@ -363,6 +384,9 @@ pub mod pallet {
 			amount: AssetAmount,
 			reason: DispatchError,
 		},
+		ChannelOpeningFeeSet {
+			fee: T::Amount,
```

Comment:
here we call it fee, but in the config we call it Flipperinos, could make it consistent

Reply:
I figured when you're setting it in the governance extrinsic, it's useful to know that it needs to be set in flipperinos. If you see an event with fee: 1,000,000,000,000, it's already obvious that it's not measured in flip.

Reply:
would be the first time exposing `flipperinos`, currently we only refer to it internally - not necessarily against it, but might not be the clearest to external readers

Reply:
I've changed it to fee - the doc comment mentions the denomination.